api/gen/swagger.yaml

```diff
@@ -49796,8 +49797,17 @@ paths:
             properties:
               address:
                 type: string
+                description: The AutoCCTP address associated with the inputs parameters.
```

Comment:
think there is an unnecessary 's` in 'inputs'

Reply:
This file is generated by via the Protobuf in the AutoCCTP repository. I'll go ahead and open a PR for that change, but this can be shipped as is for now 😄